scripts/plugins/storage/sqlite/init.sql

```diff
@@ -790,6 +801,19 @@ INSERT INTO foglamp.schedules ( id, schedule_name, process_name, schedule_type,
               );
 
 
+-- DHT11 sensor south plugin - C Code
+INSERT INTO foglamp.schedules ( id, schedule_name, process_name, schedule_type,
```

Comment:
We might not need this as well.

This entry can be done via REST API.

You can check this: scripts/extras/foglamp_update

and maybe create a script or provice a copy & paste REST API URL into the README

Reply:
Once changes are done for FOGL-1568, we would have a add_service REST API for C++ services also, then direct DB entries won't be required at all.